### PR TITLE
COST-3121 update sql to populate new ocpaws columns

### PR DESCRIPTION
**Pull request by djnakabaale**
## Jira Ticket

[COST-3121](https://issues.redhat.com/browse/COST-3121)

## Description

This change will update sql to populate the new cost columns for OCP on AWS

## Testing

1. Checkout Branch
2. Restart Koku
3. Hit endpoint or launch shell
    1. You should see ...
4. Do more things...

## Notes

...

### CODE REVIEW

**Comment by adberglund** [comment]
This is looking great. There's unfortunate timing with two PRs out at the same time impacting this area. Let's sync with @maskarb on https://github.com/project-koku/koku/pull/3990/files to use the new method for the cost columns from his PR.

**Comment by adberglund** [comment]
I went ahead and updated the terms to match the recent change linked above in #3990

**Review by adberglund — APPROVED** [review]

File: koku/masu/database/presto_sql/reporting_ocpawscostlineitem_daily_summary.sql [hunk 1]
```diff
@@ -277,6 +297,10 @@ SELECT aws.uuid as aws_uuid,
         max(nullif(aws.lineitem_currencycode, '')) as currency_code,
         max(aws.lineitem_unblendedcost) as unblended_cost,
         max(aws.lineitem_unblendedcost) * cast({{markup}} as decimal(24,9)) as markup_cost,
+        max(aws.lineitem_blendedcost) as blended_cost,
+        max(aws.lineitem_blendedcost) * cast({{markup}} as decimal(33,15)) as markup_cost_blended,
+        max(savingsplan_savingsplaneffectivecost) as savingsplan_effective_cost,
+        max(savingsplan_savingsplaneffectivecost) * cast({{markup}} as decimal(33,15)) as markup_cost_savingsplan,
```
Comment (cgoodfred) [inline koku/masu/database/presto_sql/reporting_ocpawscostlineitem_daily_summary.sql:303]: nitpick (non-blocking): we prefixed all of the other elements with `aws.` or `ocp.` to identify where they came from even if they aren't ambiguous, if we wanted to add that here we could

File: koku/masu/database/sql/aws/openshift/reporting_ocpaws_cost_summary_by_account_p.sql [hunk 2]
```diff
@@ -15,6 +15,11 @@ INSERT INTO {{schema_name | sqlsafe}}.reporting_ocpaws_cost_summary_by_account_p
     account_alias_id,
     unblended_cost,
     markup_cost,
+    blended_cost,
+    markup_cost_blended,
+    savingsplan_effective_cost,
+    markup_cost_savingsplan,
+
```
Comment (cgoodfred) [inline koku/masu/database/sql/aws/openshift/reporting_ocpaws_cost_summary_by_account_p.sql:22]: nitpick (non-blocking): we don't need this extra line here

File: koku/masu/database/presto_sql/reporting_ocpawscostlineitem_daily_summary.sql [hunk 3]
```diff
@@ -167,6 +179,10 @@ SELECT aws.uuid as aws_uuid,
...
```
Comment (cgoodfred) [inline koku/masu/database/presto_sql/reporting_ocpawscostlineitem_daily_summary.sql:185]: nitpick (non-blocking): we prefixed all of the other elements with aws. or ocp. to identify where they came from even if they aren't ambiguous, if we wanted to add that here we could

**Review by cgoodfred — APPROVED** [review]
Left a few comments about some nitpicks that are up to you if you want to address them or just merge, overall LGTM!

**Review by cgoodfred — APPROVED** [review]

**Pull request merged**